src/main/java/lotto/step2/view/InputView.java

```diff
@@ -0,0 +1,25 @@
+package lotto.step2.view;
+
+import java.util.Scanner;
+
+public final class InputView {
+
+    private static final Scanner sc = new Scanner(System.in);
+
+    private InputView() {
+        throw new AssertionError();
+    }
+
+    public static String printInputOrderPrice() {
+        System.out.println("구매 금액을 입력해주세요");
+        return sc.nextLine();
+    }
+
+    public static String printInputWinningNumber() {
```

Comment:
```suggestion
    public static List<Integer> printInputWinningNumber() {
```
String을 넘겨주다보니 도메인에서 파싱까지 하게 되는데요~
개인적으론 view에서 어느정도 파싱을 해서 넘겨주어도 좋다고 생각해요ㅎㅎㅎ